src/screens/DownloadQueue.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
fix this lint error, don't ignore it
Alternatively use styled for instead of sx if its not workable